engine/grid.py

```diff
@@ -1,6 +1,10 @@
+from platform import node
```

Comment:
Is there a reason for these new imports that I am not noticing?

Reply:
@paulhsuh for line 1 `from platform import node`
@AdityaKompella6  for line 2 `from select import select`